torch/testing/_internal/common_modules.py

```diff
@@ -771,6 +771,22 @@ def module_inputs_torch_nn_BatchNorm3d(module_info, device, dtype, requires_grad
                     forward_input=FunctionInput(make_input((0, 5, 2, 2, 2))),
                     desc='zero_batch')]
 
+def module_inputs_torch_nn_ChannelShuffle(module_info, device, dtype, requires_grad, training, **kwargs):
+    make_input = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+
+    return [
+        ModuleInput(constructor_input=FunctionInput(2),
```

Comment:
This sample input where you pass in a singleton tensor is causing the failure, seems like there is a `TORCH_CHECK` that ensures input size is at least 2 [here](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/ChanelShuffle.cpp#L36-L38)